vignettes/fixest_walkthrough.Rmd

```diff
@@ -434,27 +434,26 @@ feols(y ~ x1 | id, base_did, vcov = function(x) sandwich::vcovHC(x, type = "HC1"
 
 # Instrumental variables
 
-Two stage least square estimations can be performed by simply adding in the formula the link between the endogenous and the instrumental variables. Let's have an example.
+Two stage least square estimations can be performed by simply adding in the formula the link between the endogenous and the instrumental variables. 
+Let's have an example using the Fulton fish market data (Angrist, Grady, and Imbens 2000). 
+This data consists of daily recordings of the price and quantity of fish sold in this NYC market.
+The paper uses the sea weather as a "supply shifter" to estimate the demand elasticity.
```

Comment:
Can we say a bit more about why this is necessary? Just something as simple as we want to estimate quantity/price but are facing the well-known simultaneity problem from supply and demand being jointly determined.